src/lib/context-menu.ts

```diff
@@ -332,24 +350,48 @@ export class ContextMenu {
                           insertTableFootnote(
                             this.node,
                             this.getPos(),
+                            this.view,
                             state,
-                            dispatch
+                            dispatch,
+                            tablesFootnoteLabels
                           )
                           this.props?.popper.destroy()
                         },
                         onInsert: (notes: FootnoteNode[]) => {
```

Comment:
I assume this is not a `FootnoteNode[]` anymore but it's of type `FootnoteWithIndex[]`. am I right?

Reply:
no will keep it as is

Reply:
> I mentioned, "Maybe you mean => index ? index : inlineFootnoteIndex". I assumed that the index refers to the actual label coming from the type FootnoteWithIndex.

updated to that logic

> I commented, "I assume you only need to reorder when the index was undefined". I assumed that the index would be undefined in the case of uncited notes.

still will need to reorder, what about when there is an element before the uncited one